clients/client-python/gravitino/filesystem/gvfs_config.py

```diff
@@ -0,0 +1,15 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
```

Comment:
I think the license header should be updated.

Reply:
Done